v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -83,8 +85,9 @@ func AddSubcommand(app *clir.Cli, w io.Writer) error {
 	flags := defaultDevFlags()
 	command.StringFlag("ldflags", "optional ldflags", &flags.ldflags)
 	command.StringFlag("compiler", "Use a different go compiler to build, eg go1.15beta1", &flags.compilerCommand)
-	command.StringFlag("assetdir", "Serve assets from the given directory", &flags.assetDir)
+	command.StringFlag("assetdir", "Serve assets from the given directory instead of using the provided asset FS", &flags.assetDir)
 	command.StringFlag("e", "Extensions to trigger rebuilds (comma separated) eg go", &flags.extensions)
+	command.StringFlag("reloaddirs", "Additional directories to trigger reloads (comma separated)", &flags.reloadDirs)
```

Comment:
Why was this added? Not that I mind but I'd like to discuss it 👍

Reply:
I've written a little bit more in the commit log about the use case. The main idea was to support the following scenario, that I currently have in our application.
I'm using some simple overlayFS implementation in a `fs.FS` interface, which is provided as the assets. The base FS comes from an `embed.FS` and the overlay content from a squashFS. During compilation the baseFS is either the `embed.FS` or a `os.DirFS` depending on the build tag. In combination with this new `reloadDirs` it's still possible to have reload triggering in dev mode for this scenario and because of the `os.DirFS` in dev mode, changes will be applied during reload.

Another idea I would see to add support for such a scenario could be the following. Having some specialed `wails.FS` interface, which might optionally be implemented to wrap an `fs.FS`. This new interface would bring some support for traveling up the tree of wrapped/overlay `fs.FS` interfaces and would bring support to replace an `fs.FS` implementation in the tree. In this case wails could replace an in-tree `embed.FS` with an `os.DirFS`. 
This would be rather complex and needs some detailed discussion, and the question is if it's worth the effort. Especially if the same could be achieved with the relatively small changeset of that commit. I also think using an `fs.FS` as assets is a more advanced topic, which might refine in the future and hopefully we will see more use cases and ideas how it's getting used. This might also bring more insight what user wants and help use define a more advanced support for "auto-reload" on `fs.FS` assets.

I would fully understand if you say this option heads into an undesired direction for wails.